### PR TITLE
Grammar and Syntax improvements for Namespaces.md

### PR DESCRIPTION
**Pull request by Joelsem**
Hopefully these small improvements help a bit :)

### CODE REVIEW

File: docs/learning-frontity/namespaces.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ It's easier to understand with some examples.
 
 ### Example: `comments`
 
-Imagine a Frontity package for WordPress native comments that exports a `Comment` in its libraries. It is called `wp-comments` but its namespaces is `comments`. It may be something like this:
+Imagine a Frontity package for WordPress native comments that exports a `Comment` in its libraries. It is called `wp-comments` but its namespaces are `comments`. It may be something like this:
```
Comment (mburridge) [inline docs/learning-frontity/namespaces.md:155]: I think 'its namespaces are' should be 'its namespace is'.

File: docs/learning-frontity/namespaces.md [hunk 2]
```diff
@@ -207,7 +207,7 @@ export default {
 ```
 {% endcode %}
 
-But what if \(and now it is when this become interesting\) users don't want to use WordPress native comments but [Disqus](https://disqus.com/) comments?
+But what if \(and now it is when this becomes interesting\) users don't want to use WordPress native comments but [Disqus](https://disqus.com/) comments?
```
Comment (mburridge) [inline docs/learning-frontity/namespaces.md:210]: 'and now it is when this becomes interesting' would be better as 'and now this is where it becomes interesting' or even 'and now this is where things becomes interesting'

**Comment by juanmaguitar** [comment]
@mburridge Can you close this PR if you consider there are no really fixes in this PR over the current status of this content?

**Comment by mburridge** [comment]
Minor changes applied direct to master branch, as per my comments above.

**Pull request closed without merging**